test/test_sie.py

```diff
@@ -45,7 +45,7 @@ def test_jacobian():
     # Parameters
     z_s = torch.tensor(1.2)
     x = torch.tensor([0.5, 0.912, -0.442, 0.7, pi / 3, 1.4])
-    J = lens.jacobian_reduced_deflection_angle(thx, thy, z_s, lens.pack(x))
+    J = lens.jacobian_lens_equation(thx, thy, z_s, lens.pack(x))
```

Comment:
Do we have a test comparing the finite difference and the autograd method? At least adding a check that we can call the method and that it works woud be appropriate.

Reply:
Good idea! Adding that now